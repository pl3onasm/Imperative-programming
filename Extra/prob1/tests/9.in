21 32 18944486367