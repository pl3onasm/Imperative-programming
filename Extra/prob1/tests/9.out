13885762882630464053